### PR TITLE
Dev move folder contents one level up

### LINKED ISSUE #228: Move contents from squidDrone folder one level up
* Modify wiki for new folder structure
* Modify VSCode Settings and CMake if necessary

### PR DESCRIPTION
**Pull request by elheck**
closes #228 .
Adjusted folder structure and hardcoded paths.

### CODE REVIEW

File: squidDrone/.clang-format [hunk 1]
```diff
@@ -1,4 +0,0 @@
-Language: Cpp
-BasedOnStyle: Google
```
Comment (ThorsHamster) [inline squidDrone/.clang-format]: I'm missing this file.

File: squidDrone/.cproject [hunk 2]
```diff
@@ -1,232 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<?fileVersion 4.0.0?><cproject storage_type_id="org.eclipse.cdt.core.XmlProjectDescriptionStorage">
```
Comment (ThorsHamster) [inline squidDrone/.cproject]: I'm missing this file.

File: squidDrone/.project [hunk 3]
```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ThorsHamster) [inline squidDrone/.project]: I'm missing this file.

**Review by ThorsHamster — CHANGES_REQUESTED** [review]
Three files missing.

File: squidDrone/.project (hunk 3 above)
Comment (elheck) [reply squidDrone/.project]: Readded in [this commit](https://github.com/elheck/Squiddrone/pull/239/commits/6bf0a177d6d45301e222f00792db99885480eedb)

File: squidDrone/.cproject (hunk 2 above)
Comment (elheck) [reply squidDrone/.cproject]: Readded in [this commit](https://github.com/elheck/Squiddrone/pull/239/commits/6bf0a177d6d45301e222f00792db99885480eedb)

File: squidDrone/.clang-format (hunk 1 above)
Comment (elheck) [reply squidDrone/.clang-format]: Readded in [this commit](https://github.com/elheck/Squiddrone/pull/239/commits/6bf0a177d6d45301e222f00792db99885480eedb)

**Review by ThorsHamster — APPROVED** [review]
LGTM, thanks

**Pull request merged**